components/PlaySound/PlayButton.js

```diff
@@ -0,0 +1,34 @@
+import styled from "styled-components";
+import Image from "next/image";
+import playing from "@/public/images/playing.gif";
+
+export default function PlayButton({ handleIsPlaying, audioPlaying }) {
+  return (
+    <StyledPlayButton
+      type="button"
+      onClick={handleIsPlaying}
+      audioPlaying={audioPlaying}
+    >
+      {audioPlaying ? (
+        <Image width={50} height={16} src={playing} alt={"playing audio"} />
+      ) : (
+        <p>turn sound on</p>
```

Comment:
I think here would be ideal to have an icon like 🔇 und when click turns into 🔈
But you can do thin in a later US, for now its really great!